### PR TITLE
Pull out a while loop in an if-statement

### PR DESCRIPTION
**Pull request by pirapira**
There was a while loop in the body of an if statement, and
probably this increased the code complexity.

This commit pulls the while loop out of the if statement.
Probably this makes CodeClimate happier.

### CODE REVIEW

**Comment by pirapira** [comment]
Codecov is complaining.

I'll implement a test to touch
```
        if r['result'] == 'Fail - Unable to verify':
```

**Comment by pirapira** [comment]
The diff coverage should increase from 53.33%

File: raiden_contracts/tests/test_etherscan_verify.py [hunk 1]
```diff
@@ -204,6 +204,37 @@ def test_etherscan_verify_unknown_error():
...
+        etherscan_api = api_of_chain_id[chain_id]
+        m.post(
+            etherscan_api,
+            text='{ "status": "1",'
+            '"result" : "dummy_guid", '
+            '"message" : "" }',
+        )
+        m.get(
+            etherscan_api,
+            text='{ "status": "0",'
```
Comment (palango) [inline raiden_contracts/tests/test_etherscan_verify.py]: Would it be easier/nicer to use 
```
text="""
{...}
"""
```
here?

Comment (pirapira) [reply raiden_contracts/tests/test_etherscan_verify.py]: Yes, that's much nicer.  I'll change these in this PR.

File: raiden_contracts/tests/test_etherscan_verify.py [hunk 2]
```diff
@@ -160,9 +160,13 @@ def test_etherscan_verify_already_verified():
         etherscan_api = api_of_chain_id[chain_id]
         m.post(
             etherscan_api,
-            text='{ "status": "0",'
-            '"result" : "Contract source code already verified",'
-            '"message" : "" }',
+            text="""
```
Comment (palango) [inline raiden_contracts/tests/test_etherscan_verify.py]: From a quick ipython session it seems like you don't even need to escape the `"`:
```
In [1]: a = """
   ...: {"abc": "def"}
   ...: """

In [2]: a
Out[2]: '\n{"abc": "def"}\n'

In [3]:             text="""
   ...:             {
   ...:                 \"status\": \"0\",
   ...:                 \"result\" : \"Contract source code already verified\",
   ...:                 \"message\" : \"\"
   ...:             }
   ...:             """,

In [4]: text
Out[4]: ('\n{\n    "status": "0",\n    "result" : "Contract source code already verified",\n    "message" : ""\n}\n',)
```

Comment (pirapira) [reply raiden_contracts/tests/test_etherscan_verify.py]: My bad, not even tried this.

**Pull request merged**